resources/openapi.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.0.0
+info:
+  title: SciQuel Backend API
+  version: 1.0.0
+  description: Backend API for SciQuel Web Application
+paths:
+  /api/stories:
+    get:
+      tags:
+        - Stories
+      summary: Return a list of stories
+      description: Return 4 stories
+      responses:
+        "200":
+          description: Successfully return a list of stories
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Story"
```

Comment:
We're gonna have to change this to be paginated in the future, but for now this is fine.

Reply:
I updated API docs according to the latest implementation